src/components/app-login/app-login.tsx

```diff
@@ -1,28 +1,61 @@
-import { Component, h, Prop } from '@stencil/core';
+import { Component, h, Prop, State } from '@stencil/core';
 import { MatchResults } from '@stencil/router';
 
 @Component({
   tag: 'app-login',
-  styleUrl: 'app-login.scss',
+  styleUrls: ['app-login.scss'],
+  assetsDirs: ['../../assets'],
   shadow: true,
 })
 export class AppLogin {
   @Prop() match: MatchResults;
-  @Prop() loading = false;
+
+  @State() warningMessage = false;
```

Comment:
warningMessage sounds like its holding a message, but its boolean
so should be renamed, like showWarningMessage